### DISCUSSION TITLE
can I scroll in the list of CPUs or zoom the pane?

### DISCUSSION THREAD

**Discussion by tessus** [aristocratos/btop#516] [Q&A]
On two of my machines I have 32 CPUs (threads), but only 10 are shown, unless I close the mem, net, and proc boxes.

Is there a way to scroll in the list of CPUs, or even better zoom the cpu box so that all CPUs are shown temporarily.

**Comment by VA1DER** [comment]
Not that I know of.  But a change that might be interesting is to switch the per-CPU graphs from horizontal to vertical, and make them a single "pixel" wide each.  That would be compact, still useful, and I think quite visually appealing.  You would lose the per-cpu label, but really, for systems that have more CPUs than you can visually easily count the position of a pixel in that graph, it's not likely identifying the core number would be usefukl.

**Reply by tessus** [reply]
> it's not likely identifying the core number would be usefukl.

Well, actually it is. If I see that an application is using only a certain core or if only a subset of cores are used, even though I am using a multithreaded app (plus it is configured to use X cores, where X > than the number of cores used), there might be something off.
Also, in case I am pinning the workload to certain cores, I want to be able to verify that these cores are actually used.

**Comment by VA1DER** [comment]
Then perhaps it could be done in groups of 4 with a pixel between them.  That should make it pretty easy visually to identify the exact core in question,
EDIT: Or just make each group of four slightly different colours

**Reply by tessus** [reply]
This is also a good idea. The different color could be a visual overload, but I can't say. I suck in UI design. I leave that to the professionals.
